examples/common/src/chariott/api.rs

```diff
@@ -11,18 +11,19 @@ use std::{
     env,
 };
 
-use super::{inspection::Entry as InspectionEntry, proto, value::Value};
+use super::{inspection::Entry as InspectionEntry, value::Value};
 
 use async_trait::async_trait;
-use chariott_common::{
-    error::{Error, ResultExt as _},
-    proto::{
-        common::{
-            fulfillment::Fulfillment as ProtoFulfillment, DiscoverFulfillment, InspectFulfillment,
-            InvokeFulfillment, ReadFulfillment, SubscribeFulfillment, WriteFulfillment,
-        },
-        runtime::FulfillResponse,
+use chariott_common::error::{Error, ResultExt as _};
+use chariott_proto::{
+    common::{
+        discover_fulfillment::Service as ProtoService, DiscoverFulfillment, DiscoverIntent,
```

Comment:
nit: So many types here so may be import everything via `*`? And then perhaps the `ProtoService` exception can be move closer to where it's used/needed? For example:

```diff
diff --git a/examples/common/src/chariott/api.rs b/examples/common/src/chariott/api.rs
index 78c0a698..754f88a7 100644
--- a/examples/common/src/chariott/api.rs
+++ b/examples/common/src/chariott/api.rs
@@ -384,8 +384,10 @@ pub struct Service {
     pub schema_reference: Box<str>,
 }
 
-impl From<ProtoService> for Service {
-    fn from(value: ProtoService) -> Self {
+use chariott_proto::common::discover_fulfillment::Service as DiscoverFulfillmentService;
+
+impl From<DiscoverFulfillmentService> for Service {
+    fn from(value: DiscoverFulfillmentService) -> Self {
         Service {
             url: value.url.into(),
             schema_kind: value.schema_kind.into(),
```

Reply:
I removed the `ProtoService` exception in favor of `ServiceMessage`. I tend not to use glob imports, for reasons as described [in the Clippy lints](https://rust-lang.github.io/rust-clippy/master/index.html#wildcard_imports) and [this thread](https://mail.mozilla.org/pipermail/rust-dev/2014-March/thread.html#8981) (the argument from [Bob Ippolito](https://mail.mozilla.org/pipermail/rust-dev/2014-March/008999.html) makes sense to me).